rfcs/0136-stabilize-incrementally.md

```diff
@@ -0,0 +1,205 @@
+For example, there is no reason in theory pure eval requires Flakes, but without the ability to populate an initial whitelist of store paths that are safe to import it in practice does.
+
+This is especially noticeable for new CLI features that *previously did*, in fact, work without Flakes.
+For example, in earlier versions of Nix `nix search` worked without Flakes.
+
+## A plan all sides can be happy with
+
+Stabilizing the new CLI and Flakes will end the saga of the past few years.
+It is a last good chance to soothe some of these tensions and put us on a good foot moving forward.
+The new NixOS Foundation gives the RFC authors hope Nix is serious about addressing these sorts of governance issues, and thus would be a good way to further demonstrate we are turning a new leaf.
+
+The plan below attempts to make all sides happy, despite their seemingly irreconcilable differences.
+The basic thrust is to proceed with stabilization in small steps.
+Two benefits are envisioned:
+
+### Benefit 1: Keep discussions on track.
+
+Threads on Flakes in the past, such as that for the [original RFC](https://github.com/NixOS/rfcs/pull/49), have been impossible to follow because the sheer magnitude of different topics being discussed.
+Small stabilization steps are meant to yield *focused, bounded* discussions that actually read well.
+This also ensures any issues that do arise are actually brought to attention, and not lost within a deluge of other topics.
+
+### Benefit 2: Build trust by starting with the least controversial material.
+
+Flakes are obviously the most controversial, and so they are saved for last.
+The CLI is more broadly popular, but still is a lot of material to discuss.
+The store-only subcommands are Nix's "plumbing" as opposed to "porcelain" commands, and thus have the simplest (if not most user-friendly) purposes.
+This "dusty corner" of the new CLI is rather calm with a very constrained design space, and far less acrimony.
+
+## Conclusion
+
+By starting with this relatively easy material to stabilize, we can prove we can all come together as a community to agree on a design after all.
+This should build trust between opposing individuals and factions, giving us a foundation upon which to tackle the more challenging material in subsequent steps.
+
+Since these features became very popular while they are still unstable, there is now both an urgency to stabilize them, and little ability to modify them.
+This is the opposite of how experimental features are supposed to leisurely incubate with both plenty of flexibility to change them, and little rush to stabilize them until they feel ready.
+But, there is little we can do about this at this point, an this RFC recognizes that fact and does *not* try to resist it.
+
+But, we have crossed that Rubicon and there is no turning back; this RFC *doesn't* attempt to change Flakes or the command line.
```

Comment:
I don't know. I'm not convinced of this whole subsection because this does not align with the motivation stated before. Incremental stabilisation should instead allow room for reconsideration and systematically incorporating feedback and long-standing criticism. I read this RFC as an attempt to reconcile the experimental forays with the objection that not enough care has been taken with maintaining the core use cases, and just saying "well that train has left" doesn't fit that image at all.